lib/krikri/aggregation_entity_behavior.rb

```diff
@@ -0,0 +1,14 @@
+
+# TODO:  move to a subdirectory?
+
+module Krikri
+  class AggregationEntityBehavior < EntityBehavior
```

Comment:
I'm not sure, but the name spacing may be necessary if we arrange these subclasses into their own subdirectory, as I suggest.